python/mxnet/module/base_module.py

```diff
@@ -625,7 +625,7 @@ def output_shapes(self):
     ################################################################################
     # Parameters of a module
     ################################################################################
-    def get_params(self):
+    def get_params(self, copy_to_cpu=True):
```

Comment:
For all the places you added this new param, could you please update the comments for the function to explain what this copy_to_cpu is for, like what you did in executor_group.py? May also be good to add some explanation why it is not used here and where it is going to be used. The readers might have the same confusion why the param is in function signature but not used in the implementation.

Reply:
As we discussed offline, based on our tests of distributed training on 1 node and 2 node p3.16xlarge instances, the training accuracy looks normal. We also saved and compared the module parameters from a rank from each node in the 2 node scenario and they look identical. I think we can remove this copy_to_cpu argument change